runtime/inspector.rs

```diff
@@ -275,11 +276,14 @@ async fn server(
             (&http::Method::GET, path) if path.starts_with("/ws/") => {
               handle_ws_request(req, inspector_map.clone())
             }
+            (&http::Method::GET, "/json/version") => {
+              handle_json_version_request(json_version_response.clone())
+            }
             (&http::Method::GET, "/json") => {
               handle_json_request(inspector_map.clone())
             }
-            (&http::Method::GET, "/json/version") => {
-              handle_json_version_request(json_version_response.clone())
+            (&http::Method::GET, "/json/list") => {
+              handle_json_request(inspector_map.clone())
```

Comment:
Looks like this route has been inadvertently removed when rewriting inspector server. @lperlaki please add a test case in `cli/tests/integration_tests.rs` that hits this endpoint

Reply:
@bartlomieju I wil do that but I couldn't find how any other integration test that did http request, what library/method should I use?

Reply:
@lperlaki you can use `reqwest` for that purpose